_i18n/en.yml

```diff
@@ -147,30 +147,30 @@ merchants:
-  swapsdescr: A new exciting way to exchange Monero is Atomic Swaps, which <a href="https://www.getmonero.org/2021/08/20/atomic-swaps.html">we announced</a> on August 2021. Atomic Swaps are a completely decentralized way to swap Bitcoin <-> Monero (more info in the blog post). The technology is brand new, so there aren't many swap providers yet, but you can already sell XMR for BTC using this protocol at
+  swapsdescr: A new exciting way to exchange Monero is Atomic Swaps, which <a href="https://www.getmonero.org/2021/08/20/atomic-swaps.html">we announced</a> on August 2021. Atomic Swaps are a completely decentralized way to swap Bitcoin <-> Monero (more info in the blog post).
   bisqdescr: Decentralized P2P exchange based on Bitcoin and Tor.
   cardfoss: "Open source:"
   cardkyc: "<b>No KYC</b> (Know Your Customer):"
   cardbtcfiat: (BTC &#8596; fiat possible)
   cardonion: "Onion address:"
   cardi2p: "I2P address:"
   noncustodial: "Non-custodial:"
-  hodlhodldescr: P2P trading platform with non-custodial escrow.
   locmondescr: Online P2P exchange offering person-to-person trades.
   depsell: Depends by the seller
   locnojs: (but no-JS version available)
   simpleswapdescr: Non-custodial instant exchange.
   disclaimer: |
     "Please note: these links are being provided as a convenience and for informational purposes only; they do not constitute an endorsement by the Monero community of any products, services or opinions of the corporations or organizations or individuals listed. The Monero community bears no responsibility for the accuracy, legality, or content of these external sites. Contact the external site for answers to questions regarding its content. As always, caveat emptor ('buyer beware'); you are responsible for doing your own research. Always use judgement when making online purchases."
   visitbisq: Visit Bisq
-  visithodlhodl: Visit HodlHodl
   centrexchanges: Centralized exchanges (CEXs) & Swappers
   centrexchangesp: If you prefer to use centralized exchanges, here is a list of renowned CEXes and swappers. Many more exchanges support Monero, we list here only a few reputable ones.
   cexp: Centralized exchanges that offer exchanging Monero for national currencies and cryptocurrencies.
   swappersp: Swappers let users exchange XMR for other cryptocurrencies.
   basicswapdexdesc: Decentralized P2P atomic swaps with a strong focus on the user experience.
   visitbasicswapdex: Visit BasicSwapDEX
   setreq: Setup required
+  unstoppableswapdesc: Exchange Bitcoin for Monero using Atomic Swaps with our desktop software. No registration or middleman required. Over the Tor network. Accessible to everyone. Community funded.
```

Comment:
``No registration or middleman required. Over the Tor network.`` is unnecessary, its already mentioned in the info chart

Reply:
Could be shortened to "Exchange bitcoin for monero using atomic swaps"